### PR TITLE
feat(lb): add support for subscriber

### PR DESCRIPTION
**Pull request by alekc**
Misses: docs, integration with lb

Related: https://github.com/terraform-providers/terraform-provider-scaleway/issues/424

### CODE REVIEW

File: scaleway/resource_lb_subscriber_beta.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+							Required:    true,
+							Description: "Email who receive alert.",
+						},
+					},
+				},
+			},
+			"webhook_config": {
+				ConflictsWith: []string{"email_config"},
+				MaxItems:      1,
+				Description:   "WebHook URI configuration. Only one of email_config and webhook_config may be set.",
```
Comment (QuentinBrosse) [inline scaleway/resource_lb_subscriber_beta.go]:
```suggestion
				Description:   "WebHook URI configuration.",
```

File: scaleway/resource_lb_subscriber_beta.go [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+				Type:          schema.TypeList,
+				Optional:      true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"uri": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "URI who receive POST request.",
```
Comment (QuentinBrosse) [inline scaleway/resource_lb_subscriber_beta.go]: URI receives the POST request.
```suggestion
							Description: "URI receiving the POST request.",
```

**Comment by alekc** [comment]
@QuentinBrosse  Applied your suggestions. I think it makes more sense to put everything in this PR since subscriber is useless without the support from LB.  I'll update this pr soon.

File: scaleway/provider.go [hunk 3]
```diff
@@ -89,6 +89,7 @@ func Provider(config *ProviderConfig) plugin.ProviderFunc {
 				"scaleway_lb_certificate":                      resourceScalewayLbCertificate(),
 				"scaleway_lb_frontend":                         resourceScalewayLbFrontend(),
 				"scaleway_lb_route":                            resourceScalewayLbRoute(),
+				"scaleway_lb_subscriber":                       resourceScalewayLbSubscriber(),
```
Comment (Monitob) [inline scaleway/provider.go:92]: Can we add documentation for this please.

**Pull request closed without merging**